main.go

```diff
@@ -76,6 +77,17 @@ func main() {
 	logger.Println("Finished preparing included app")
 	defer os.RemoveAll(appPath)
 
+	var tcpAppPath string
+	if cfg.OptionalTests.RunTcpAvailability {
+		tcpAppPath, err = prepareIncludedApp("tcpApp", tcpApp.Source)
+		if err != nil {
+			logger.Println("Failed to prepare included tcpAapp: ", err)
+			performMeasurements = false
+		}
+		logger.Println("Finished preparing included tcpApp")
```

Comment:
Do you mind just using `...included tcp app"` so it looks similar to the syslog sink app case?

Reply:
👍